bot/tests/test_artifacts.py

```diff
@@ -72,8 +72,9 @@ def add_dir(files):
         'linux_xpcshell-7_code-coverage-grcov.zip'
     ])
 
-    with pytest.raises(Exception, message='suite and chunk can\'t both have a value'):
+    with pytest.raises(Exception) as e:
```

Comment:
You can use `match` instead of `message`

Reply:
Done.